library/src/main/java/com/novoda/downloadmanager/MigrationJob.java

```diff
@@ -53,6 +56,86 @@ public void run() {
         onUpdate(migrationStatus);
 
         Log.d(TAG, "about to extract migrations, time is " + System.nanoTime());
+        Log.d(TAG, "partial migrations are all EXTRACTED, time is " + System.nanoTime());
```

Comment:
Shouldn't this be logged out after `migratePartialDownloads()` (just a couple lines down 😛  )